drsm/api.go

```diff
@@ -18,7 +18,7 @@ type DbInfo struct {
 
 type PodId struct {
 	PodName     string `bson:"podName,omitempty" json:"podName,omitempty"`
-	PodInstance string `bson:"podInstance,omitempty" json:"podName,omitempty"`
+	PodInstance string `bson:"podInstance,omitempty" json:"podInstance,omitempty"`
```

Comment:
Is this not used anywhere? I need to check carefully how Amf and Smf using these values

Reply:
I did not see it being used as a `JSON` tag anywhere in the 5GC code. Feel free to take a closer look at it.
FYI, here is how the `amf` and `smf` are using this value:

amf/util/init_context.go
```go
func InitDrsm() (drsm.DrsmInterface, error) {
	...
	logger.UtilLog.Infof("NfId Instance: %v", context.AMF_Self().NfId)
	podId := drsm.PodId{PodName: podname, PodInstance: context.AMF_Self().NfId, PodIp: podip}
	logger.UtilLog.Debugf("PodId: %v", podId)
	...
```

smf/context/context.go
```go
func (smfCtxt *SMFContext) InitDrsm() error {
	...
	podId := drsm.PodId{PodName: podname, PodInstance: smfCtxt.NfInstanceID, PodIp: podip}
	...
```